databaseserviceexample.gitlab-ci.yml

```diff
@@ -0,0 +1,45 @@
+include: 
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/belt-node-complexity-analysis-ci.yml
```

Comment:
Might be useful to comment each of these includes for people (like me) who just like to read examples.